imperator-tiger/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "imperator-tiger"
+version = "0.6.1"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tiger-lib = { path = "..", features = ["imperator"] }
```

Comment:
Should now have `version = "0.8.1",` and `default-features = false,` to turn off the default ck3 feature